python/src/opendp/extras/numpy/_make_np_clamp/__init__.py

```diff
@@ -77,4 +78,5 @@ def _function(arg):
 
 # generate then variant of the constructor
 # TODO: Show this in the API Reference?
-then_np_clamp = register_transformation(make_np_clamp)
+register(make_np_clamp)
+then_np_clamp = to_then(make_np_clamp)
```

Comment:
Does `then_np_clamp` need to be registered?

Reply:
Nope, just make. The Context API papers over this distinction.